source/package-lock.json

```diff
@@ -2711,6 +2712,20 @@
       "integrity": "sha1-x57Zf380y48robyXkLzDZkdLS3k=",
       "dev": true
     },
+    "node_modules/aws-cdk": {
+      "version": "2.20.0",
```

Comment:
use inconsistent cdk version, 2.20.0 and 2.18.0

Reply:
now upgrade cdk to the latest version  2.20.0